spec.html

```diff
@@ -30110,19 +30110,47 @@ <h1>Function.prototype.toString ( )</h1>
           1. Let _func_ be the *this* value.
           1. If Type(_func_) is Object and _func_ has a [[SourceText]] internal slot and _func_.[[SourceText]] is a sequence of Unicode code points and HostHasSourceTextAvailable(_func_) is *true*, then
             1. Return CodePointsToString(_func_.[[SourceText]]).
-          1. If _func_ is a <emu-xref href="#sec-built-in-function-objects">built-in function object</emu-xref>, return an implementation-defined String source code representation of _func_. The representation must have the syntax of a |NativeFunction|. Additionally, if _func_ has an [[InitialName]] internal slot and _func_.[[InitialName]] is a String, the portion of the returned String that would be matched by |NativeFunctionAccessor?| |PropertyName| must be the value of _func_.[[InitialName]].
+          1. If _func_ is a <emu-xref href="#sec-built-in-function-objects">built-in function object</emu-xref>, return an implementation-defined String source code representation of _func_. The representation must have the syntax of a |NativeFunction|. Additionally, if _func_ has an [[InitialName]] internal slot and _func_.[[InitialName]] is a String, then calling NativeFunctionName with the returned String as the argument must return the value of _func_.[[InitialName]].
```

Comment:
Even if the normative change is rejected, we should probably keep these structural changes because the current text is incoherent in multiple ways:
1. It doesn't seem correct to talk about something being matched by a _sequence_ of grammar nonterminals.
2. Grammar nonterminals match against sequences of code points, not against Strings.
3. When _func_.[[InitialName]] is an _empty_ String and the returned value is like `"function () {…}"`, then wouldn't the matched portion be undefined and therefore fail to match it?